lib/configuration/variables/sources/merge.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const ServerlessError = require('../../../serverless-error');
+
+module.exports = {
+  resolve: ({ params }) => {
+    if (!Array.isArray(params) || params.length < 2) {
```

Comment:
Let's simplify check to `!params || params.length < 2` (`params` if exists is guaranteed to be an array)